packages/components/pie-modal/src/index.ts

```diff
@@ -574,10 +615,11 @@ export class PieModal extends PieElement implements ModalProps {
             aria-live="polite"
             aria-busy="${isLoading ? 'true' : 'false'}"
             data-test-id="pie-modal">
+            ${this.renderModalImageSlot()}
             <header class="c-modal-header" data-test-id="modal-header">
                 ${this.renderBackButton()}
```

Comment:
This may be coming in a different PR but will the back button also move onto the image?